basicMysql/mysqlClass.php

```diff
@@ -0,0 +1,280 @@
+<?php
+  //vendor is folder realted to composer and here used for 'Guzzle'
+  //require("../vendor/autoload.php");
+  use PHPMailer\PHPMailer\PHPMailer;
+
+  //Client is a class which coming from GuzzleHttp and Guzzle used it for email validation.
+  use GuzzleHttp\Client;
+  class MysqlMethods
+  {
+    //Login user-name
+    public $getname;
+    //Login user-password
+    public $getpwd;
+    //Login user-name error message
+    public $status_name = false;
+    //Login user-password error message
+    public $status_pwd = false;
+
+    /**
+    * function getLogin($username, $userpwd)
+    *
+    * @param [string] $username : This contains the user-name.
+    * @param [string] $userpwd : It contains user-password.
+    * This function first check user-name and password from database and then, if data are valid then return true otherwise return false.
+    *
+    * @return boolean
+    */
+    function getLogin($username, $userpwd){
+      $this->getname = $username;
+      $this->getpwd = $userpwd;
+
+      $conn = new mysqli("localhost", 'root', 'Abhi4531@my', 'User_DB');
+      //$sql = "SELECT userName, userPwd FROM User WHERE userName = '$username' AND userPwd = '$userpwd'";
+      $sql = "SELECT userName, userPwd FROM User WHERE userName = '$username'";
+
+      $result = $conn->query($sql);
+
+      if ($result->num_rows > 0) {
+        while($row = $result->fetch_assoc()) {
+          if($userpwd === $row["userPwd"]){
+            return true;
+          }
+        }
+        $this->status_pwd = true;
+        return false;
+      }
+      else {
+        $this->status_name = true;
```

Comment:
Actually `$status_name` is boolean type varaible which is only responsible for showing the error message like if username is valid then error messgae should not show in that case `$status_name` is false and simillarly if username is invalid then `$status_name` is true just for showing the error messsage to user.

Reply:
Then could you name it as an error message or something similar to an error I thought the status means the current condition of the name in the database.

Reply:
I added a proper comment for this variable please check once
`//It is checking the user-name is valid or not, if valid then false otherwise true
public $status_name = false;`
I did't change beacuse I think name is meaningful like status of user-name is valid or not. now is ot okay?

Reply:
As you mentioned earlier `$status_name ` was checking for an error that's the reason I asked you to rename it but now you are saying it is checking the status, could you clarify it cause it is a bit difficult for me to understand what `$status_name` is?

Reply:
It is a `boolean` type when userName is invalid then it will `true` and if valid then it will be `false', after that according to the value of `$status_name` if it if false then error message will not come and if true then error message will come to user.

Reply:
Understood your point